projects/iio_demo/pinmux_config.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
commit description is longer than 75 characters/line.

Reply:
done